functional-tests/src/support/action/addOrganisationName.ts

```diff
@@ -0,0 +1,32 @@
+import {waitForDisplayed} from '@nice-digital/wdio-cucumber-steps/lib/support/action/waitForDisplayed'
+import {clickElement} from '@nice-digital/wdio-cucumber-steps/lib/support/action/clickElement';
+import selectors from '../selectors';
+import {setInputField} from '@nice-digital/wdio-cucumber-steps/lib/support/action/setInputField';
+
+
+
+export async function addOrganisationName(name: string): Promise<void> {
```

Comment:
lots of awaits...any value in a try catch around these to cover any errors?

Reply:
maybe try catch isn't appropriate for functional test, not sure.

Reply:
I don't think it is neccessary within the tests as we get error information back from wdio.

Reply:
Fair enough thanks Martin